### PR TITLE
Adjust TableElement content order

### PR DESCRIPTION
**Pull request by IslamJMomani**
Based on the designs, the table element footer should be displayed directly after the table and before the caption.

Screenshot 2024-01-11 at 11 59 42 AM

### CODE REVIEW

**Review by mbetamony — APPROVED** [review]

**Comment by IslamJMomani** [comment]
@mbetamony, thank you for the code review. Do the snapshot changes make sense to you? I can't explain them.

**Comment by mbetamony** [comment]
Good catch, I didn't actually look at the tests before (apologies) 
Removing this would probably fix it https://github.com/Atypon-OpenSource/manuscripts-transform/blob/51b7a7be64373cae3abe82bd2eb1a533f10aac6c/src/jats/importer/parse-jats-article.ts#L125 

I'll add Hashem as a reviewer as well for confirmation

File: src/schema/nodes/table_element.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export interface TableElementNode extends ManuscriptNode {
 
 export const tableElement: NodeSpec = {
   content:
-    '(table | placeholder) figcaption? table_element_footer? (listing | placeholder)',
+    '(table | placeholder) table_element_footer? figcaption? (listing | placeholder)',
```
Comment (hhusban) [inline src/schema/nodes/table_element.ts:41]: This order should also be reflected to the dom being passed to the body-parser

**Comment by IslamJMomani** [comment]
> Good catch, I didn't actually look at the tests before (apologies) Removing this would probably fix it
> 
> https://github.com/Atypon-OpenSource/manuscripts-transform/blob/51b7a7be64373cae3abe82bd2eb1a533f10aac6c/src/jats/importer/parse-jats-article.ts#L125
> 
> I'll add Hashem as a reviewer as well for confirmation

You were right, removing it fixed that problem. Thank you both, @hhusban and @mbetamony, for your guidance.

**Review by hhusban — APPROVED** [review]

**Review by mbetamony — APPROVED** [review]

**Comment by mbetamony** [comment]
> > Good catch, I didn't actually look at the tests before (apologies) Removing this would probably fix it
> > https://github.com/Atypon-OpenSource/manuscripts-transform/blob/51b7a7be64373cae3abe82bd2eb1a533f10aac6c/src/jats/importer/parse-jats-article.ts#L125
> > 
> > I'll add Hashem as a reviewer as well for confirmation
> 
> You were right, removing it fixed that problem. Thank you both, @hhusban and @mbetamony, for your guidance.

You're most welcome.

**Pull request merged**